src/test/resources/inventory-no-sku.json

```diff
@@ -3,8 +3,6 @@
   "version": 1,
   "createdAt": "2016-02-12T14:34:57.099Z",
   "lastModifiedAt": "2016-02-12T14:34:57.099Z",
-  "supplyChannel": {
```

Comment:
Could not get the reason for the changes here ?

Reply:
This `{}` brackets are considering(file read operation) as null values for the object. So in this case we have one object with all the values null. So, Either the supplyChannel should be null or to be removed.
Example:
supplyChannel: {typeId: null, id: null, obj: null }

Reply:
Now when I added this value back it goes to a null pointer exception in the test, which would be nice to bypass as before. I could also say that this is a bad input and not a valid response from the platform in case of project-to-project sync so we could skip this comment. 

But in general, in the first place, I would expect improving something is better than omitting the potential errors with changing test fixtures. (here I am not sure where NPE comes could be the mocks are not configured with the new scenario)

Reply:
We are getting NPE while calling the containsKey method from the cache. Since the id is null here in this case. Yes, This input is not valid. We have to handle validations for this. Can we create a ticket for this and do some investigation and add few more validations for such inputs?

Reply:
If you know the input is valid in terms of platform, no. As the source data comes from API.